privacyidea/lib/eventhandler/webhookeventhandler.py

```diff
@@ -130,8 +130,12 @@ def do(self, action, options=None):
         if request is not None:
             token_serial = request.all_data.get('serial')
             tokenowner = self._get_tokenowner(request)
-        user = User(login=g.logged_in_user.get('username'),
-                    realm=g.logged_in_user.get('realm'))
+
+        try:
+            user = User(login=g.logged_in_user.get('username'),
+                        realm=g.logged_in_user.get('realm'))
+        except:
```

Comment:
using a broad `except` statement might catch errors that we wish to propagate. Can You determine which error occurs if there is no logged in user?
The `except` clause should be something like:
```python
try:
    ...
except UserError as _e:
    user = None
```
Of course the error should be appropriate of what we want to catch here.

Also is there a possibility to test this?